src/sudo-prompt/build-sudo-prompt

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# defaults domain must be an absolute path
```

Comment:
What does `defaults domain` refer to here? That sounds like a `/usr/bin/defaults` thing, but that's not referenced in this script.

Reply:
The script used `defaults` before I just changed it to `plutil`. Removed the comment. `plutil` probably doesn't need the absolute path, but it doesn't hurt.